library/src/scala/tasty/reflect/Printers.scala

```diff
@@ -1140,9 +1146,19 @@ trait Printers
         }
       }
 
-      def printTargDef(arg: TypeDef, isMember: Boolean = false): Buffer = {
-        this += arg.name
-        arg.rhs match {
+      def printTargDef(arg: (TypeDef, TypeDef), isMember: Boolean = false, isDef:Boolean = true): Buffer = {
```

Comment:
Maybe a document for `printTargDef `, explaining what the meaning of `(TypeDef, TypeDef)`, the same for `printTargsDefs` as well,